File: ApplicationCore/Entities/Expenditure.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ApplicationCore.Entities
{
    public class Expenditure
    {
        public int Id { get; set; }
        public int? UserId { get; set; }

        public decimal Amount { get; set; }

        public string Description { get; set; }

        public DateTime? ExpDate { get; set; }

        public string Remarks { get; set; }

        public User User { get; set; }
    }
}

File: AndyZhang-BudgetTrackerSPA/src/app/shared/models/Expenditure.ts

export interface Expenditure {
    id:          number;
    userId:      number;
    amount:      number;
    description: string;
    expDate:     string;
    remarks:     string;
}

export interface NewExpenditure {
    userId:      number;
    amount:      number;
    description: string;
    expDate:     string;
    remarks:     string;
}